src/organisms/sensors.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, size):
     def sight(self, config, world, x_pos, y_pos):
         view = []
         for row in range(-self.size, self.size + 1, 1):
-            for tile in range(-self.size, self.size + 1, 1):
+            for tile in range(-self.size + abs(row), self.size + 1 - abs(row), 1):
```

Comment:
Could you add a comment here? I'm not sure if _one can grasp that at first glance_.